_about/notes/0.5.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Alpha2 Traffic Rules**. Add support for an updated model of routing and destination rules. TODO: Doc link?
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
+
+- **Automatic Proxy Injection**. Use new Kubernetes 1.9 mutating webhook functionality to provide automatic
```

Comment:
We should note that auto side car injection will not work for k8s 1.7 and 1.8.

Reply:
Done.